modules/social_features/social_group/src/SocialGroupHelperService.php

```diff
@@ -324,4 +324,59 @@ public function addMemberFormField() {
     ];
   }
 
+  /**
+   * Returns titles list of all groups, ordered by their type and/or label.
+   *
+   * @param bool $split
+   *   (optional) TRUE if groups should be split by type. Defaults to FALSE.
+   *
+   * @return array
+   *   Array of group ids and group labels.
+   */
+  public static function getGroups(bool $split = FALSE): array {
+    $split_lower = strtolower((string) $split);
+    if (!empty($data = &drupal_static("_social_user_get_groups_{$split_lower}", []))) {
```

Comment:
function name changed, and now is "getGroups", we should change this unique key, how about: 

`social_group_helper_service_get_groups`

Reply:
Updated.